### DISCUSSION TITLE
Information on newton raphson load entries

### DISCUSSION THREAD

**Discussion by lschuler1** [pyansys/pymapdl-reader#145] [Q&A]
Hi all,

I am trying to retrieve the residual Newton-Raphson load vector from the `.emat` file.
To what exactly the "newton raphson load entries" correspond, when using the command `emat_file.read_element()`?

I obtain some values in the order of 10^300 which are very high compared to the load vector that I can also obtain from the `.full` file.

Thank you,
Louis

**Comment by akaszynski** [comment]
> I obtain some values in the order of 10^300 which are very high compared to the load vector that I can also obtain from the .full file.

Depending on the version of MAPDL that generated the result file, it's quite likely that the entries are now corrupted as there's been a change in the API.

@PProfizi, do we support reading *.emat files within https://github.com/pyansys/pydpf-core?

**Comment by PProfizi** [comment]
Hi @akaszynski. Sadly it seems we do not yet support *.emat files.
